From ba840b5350971217976de34730a3a89a222a1b77 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 4 May 2016 14:39:57 +0800
Subject: [PATCH 1045/5242] MLK-12757-2 ARM: dts: imx6ull-ddr3-arm2.dts: add
 usdhc2 support

commit  77aba73e925fbd153d8b2b56b40cb89f17546777 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add usdhc2 support, due to cd/wp pin conflict with usdhc1, this
patch drop these two pins, and make usdhc2 as no removeable.
Moreover, due to VSELECT pin is not connected by default, we also
add no-1-8-v property.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index dd29e61..b0557a9 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -996,10 +996,10 @@
 	pinctrl-0 = <&pinctrl_usdhc2>;
 	pinctrl-1 = <&pinctrl_usdhc2_100mhz>;
 	pinctrl-2 = <&pinctrl_usdhc2_200mhz>;
-	cd-gpios = <&gpio1 19 GPIO_ACTIVE_LOW>;
-	wp-gpios = <&gpio1 18 GPIO_ACTIVE_HIGH>;
+	non-removable;
+	no-1-8-v;	/* VSELECT not connected by default */
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	vmmc-supply = <&reg_sd2_vmmc>;
-	status = "disabled";
+	status = "okay";
 };
-- 
1.7.9.5

